proxy/logging/Log.cc

```diff
@@ -1150,6 +1156,11 @@ Log::create_threads()
   ink_atomiclist_init(flush_data_list, "Logging flush buffer list", 0);
   Continuation *flush_cont = new LoggingFlushContinuation(0);
   eventProcessor.spawn_thread(flush_cont, "[LOG_FLUSH]", stacksize);
+
+  {
+    std::unique_lock<std::mutex> lk(flush_condition_mutex);
+    flush_condition_variable.wait(lk, [] { return flush_thread_is_initialized; });
+  }
```

Comment:
I don't think we know if this is the only thread that can cause this problem.  May be for now, but this could bite us, if in the future we add more threads that start before plugin initialization.

Reply:
Right. It isn't the only thread. But the flushing thread doesn't go through the UnixEventProcessor.cc DbgCtl::update() logic that's also in this patch. Thus we ensure that we log here early before the dlopen calls for plugins.